azure-iot-device/tests/common/transport/mqtt/test_mqtt_transport.py

```diff
@@ -11,7 +11,7 @@
 from azure.iot.device.common.transport.mqtt.mqtt_transport import MQTTTransport
 from azure.iot.device.common.transport import constant
 from azure.iot.device.iothub.auth.authentication_provider_factory import from_connection_string
-from mock import MagicMock, patch
+from mock import MagicMock, patch, ANY
```

Comment:
lets stop using mock altogether tbh. the mocker fixture from pytest-mock is syntactically smoother.

Reply:
OK

---
In reply to: [279687589](https://github.com/Azure/azure-iot-sdk-python-preview/pull/87#discussion_r279687589) [](ancestors = 279687589)